chuxmykel/politico
# #163522847 Make all links work

#### What does this PR do?
- Made all links functional
- Made create/edit and delete links on admin page function.
#### Description of Task to be completed?
- Fix all links

#### How should this be manually tested?
- Clone the repo
- Open the pages on a browser
- Click on all links to see them work.

#### Any background context you want to provide?
- None

#### Screenshots (if appropriate)
- None
